e2e/tests/home.spec.ts

```diff
@@ -95,10 +95,11 @@ test("remove a network", async ({ page }) => {
   await page.locator('[aria-label="edit network"]').first().click()
   const removeBtn = page.locator("data-testid=remove network button")
   expect(removeBtn).toBeDisabled()
+  const name = await page.locator('input[name="name"]').inputValue()
   const url = await page.locator('input[name="url"]').inputValue()
   await page.locator('input[name="deleteUrl"]').fill(url as string)
   expect(removeBtn).not.toBeDisabled()
   await removeBtn.click()
   const networkName = await homePage.networkMenuTrigger.textContent()
-  expect(networkName).toBe("no network selected")
+  expect(networkName).not.toBe(name)
```

Comment:
The test assumed that no network would be available after deleting one. I changed it to confirm that the deleted network was no longer active.